### PR TITLE
KDE Dialog Support

### PR DESCRIPTION
**Pull request by kemzops**
#### Why?
- I had some issues with `zenity` (some of them visible in the video).
- I use KDE Plasma DE, It make sense to use `QT` Filepicker instead of `GTK`.

https://github.com/lite-xl/lite-xl-plugins/assets/52936496/718df15d-e459-4112-b3b3-c23766ff5a4d

### CODE REVIEW

File: plugins/gui_filepicker.lua [hunk 1]
```diff
@@ -1,143 +1,188 @@
...
-		-- Remove final newline zenity adds
-		abs_path = string.match(abs_path, "^[^\n]+") or ""
-		if #abs_path == 0 then
-			return
-		end
-
-		callback(abs_path)
-	end)
+config.plugins.gui_filepicker = common.merge({
+    dialogUtility = "zenity",
```
Comment (Guldoman) [inline plugins/gui_filepicker.lua:15]: Let's use snake case:
```suggestion
    dialog_utility = "zenity",
```

File: plugins/gui_filepicker.lua [hunk 2]
```diff
@@ -1,143 +1,188 @@
...
+        flags = kdialogFlags
+    end
+
+    local proc, proc_err = process.start({
+        utility,
+        table.unpack(flags)
+    })
+
+    if not proc then
+        core.error("Unable to run " .. utility .. ": %s", proc_err)
```
Comment (Guldoman) [inline plugins/gui_filepicker.lua:46]:
```suggestion
        core.error("Unable to run %s: %s", utility, proc_err)
```

**Review by Guldoman — COMMENTED** [review]
Other than a couple of nits (and the fact that this was converted from tabs to spaces :angry:) this looks good!

File: plugins/gui_filepicker.lua (hunk 2 above)
Comment (kemzops) [reply plugins/gui_filepicker.lua:46]: i mean... OK.

**Comment by kemzops** [comment]
@Guldoman i mean... `tabs` is overrated lol, ready to be merged!.

**Pull request merged**